lib/evmone/tracing.cpp

```diff
@@ -135,8 +135,9 @@ class InstructionTracer : public Tracer
             m_out << "null";
         else
             m_out << '"' << result.status_code << '"';
-        m_out << R"(,"gas":)" << result.gas_left;
-        m_out << R"(,"gasUsed":)" << (ctx.start_gas - result.gas_left);
+        m_out << R"(,"gas":)" << std::hex << "0x" << result.gas_left;
+        m_out << R"(,"gasUsed":)" << std::hex << "0x" << (ctx.start_gas - result.gas_left)
+              << std::dec;
         m_out << R"(,"output":")" << evmc::hex({result.output_data, result.output_size}) << '"';
```

Comment:
Apparently this field is not an even listed as optional in the EIP, @chfast.

Reply:
I can remove it and check once again if we have it implemented according to spec, but let's first confirm with @chfast

Reply:
It is useful for debugging. Let's keep it.

Reply:
@chfast Approve please if no other comments.

Reply:
Can one of you open a PR to the EIP adding `output` as an optional field?